huion_keys.py

```diff
@@ -22,6 +22,25 @@
 DIAL_MODES = {} 
 
 def main():
+    #Commandline arguments processing
+    parser = argparse.ArgumentParser(
+            description='Linux utility to create custom key bindings for the Huion Kamvas Pro (2019), Inspiroy Q620M, and potentially other tablets.')
+    parser.add_argument('--rules', action='store_true', default=False,
+                    help='print out the udev rules for known tablets and exit')
+    parser.add_argument('-c', '--config', type=str,
+                    help='location of config file, ~/.config/huion_keys.conf by default')
+    args = parser.parse_args()
+    if args.rules:
+        make_rules()
+        os._exit(0)
```

Comment:
@NeoTheFox  Using `os._exit` here causes a bug where the rules are not yet flushed to stdout, but the process suddenly exits. Changing it to `sys.exit` should fix it because `sys.exit` will exit after cleaning up, flushing buffers, etc.

e.g.
`python huion_keys.py --rules` -> prints rules
`python huion_keys.py --rules > foobar.txt` -> creates blank file because output is not flushed

Might not always reproduce because stdout might get flushed before `os._exit`, but it happens consistently on my system.

Reply:
Yeah, that's why I used `python -u` in the example, it makes all the output unbuffered, didn't want to introduce another import